src/main.ts

```diff
@@ -0,0 +1,123 @@
+import * as core from '@actions/core'
+import * as exec from '@actions/exec'
+import * as tc from '@actions/tool-cache'
+import * as fs from 'fs'
+
+const upToolname = 'up'
+const currentVersionUrl = 'https://cli.upbound.io/stable/current/version'
```

Comment:
This should interpolate the `channel` in place of `stable` so that we get the right current version on non-stable channels.

Reply:
ah right good catch